CleverCloud/clever-components
# deprecate accessible-name in favor of a11y-name

Fixes #893

## What does this PR do?

* 4 commits for `accessible-name` => `a11y-name`
  * cc-button
  * cc-icon
  * cc-img
  * cc-popover
* 1 commit for  `icon-accessible-name` => `icon-a11y-name`
  * cc-badge
* 1 commit for using the new `a11yName` everywhere in our codebase

## How it does?

* add property `a11yName` (and keep the `accessibleName` one)
* replace all usage of `this.accessibleName` by `this.a11yName`
* add getter/setter on `accessibleName` (but use this.a11uName inside them)
* add JSDoc about deprecation on the setter

## How to review?

* check the code
* check the storybook doc for
  * cc-button
  * cc-icon
  * cc-img
  * cc-popover
  * cc-badge
* check stories (especially on `cc-button` and `cc-icon` which have a specific story about accessible name)

2 reviewers should be enough.
